### PR TITLE
Migrate `atom.video` to Playwright

### PR DESCRIPTION
**Pull request by arelra**
## What does this change?

Migrate `atom.video` to Playwright

### CODE REVIEW

File: dotcom-rendering/playwright/tests/parallel-5/atom.video.e2e.spec.ts [hunk 1]
```diff
@@ -0,0 +1,458 @@
...
+			// user has not given consent for any purpose
+			nonPersonalizedAdMatch = adsConfig.nonPersonalizedAd;
+			// cmpGvcd = consentState.tcfv2.addtlConsent
+			cmpGvcdMatch = adTagParameters.cmpGvcd === '1~';
+		} else {
+			// user has not given consent for any purpose
+			nonPersonalizedAdMatch = !adsConfig.nonPersonalizedAd;
+			// cmpGvcd = consentState.tcfv2.addtlConsent
+			cmpGvcdMatch = !isUndefined(adTagParameters.cmpGvcd);
+		}
```
Comment (mxdvl) [inline dotcom-rendering/playwright/tests/parallel-5/atom.video.e2e.spec.ts:91]: I think there’s a few leftover comments?

Comment (arelra) [reply dotcom-rendering/playwright/tests/parallel-5/atom.video.e2e.spec.ts:91]: Those are carried over as a pointer to where the GDPR properties we send to YouTube come from in the `ConsentState` object.

File: dotcom-rendering/playwright/tests/parallel-5/atom.video.e2e.spec.ts [hunk 2]
```diff
@@ -0,0 +1,458 @@
...
+		await loadPage(
+			page,
+			'/Article/https://www.theguardian.com/uk-news/2020/dec/04/edinburgh-hit-by-thundersnow-as-sonic-boom-wakes-residents',
+		);
+		await cmpAcceptAll(page);
+
+		await waitForIsland(page, 'YoutubeBlockComponent');
+
+		// Make sure overlay is displayed
+		const overlaySelector = `[data-testid^="youtube-overlay-S0CE1n-R3OY"]`;
```
Comment (mxdvl) [inline dotcom-rendering/playwright/tests/parallel-5/atom.video.e2e.spec.ts:133]: There's a few places where we use this magic string for the video ID. Is it worth extracting to a variable?

**Review by mxdvl — APPROVED** [review]
LGTM — assuming this is mostly a port.

**Pull request merged**